### PR TITLE
Minecraft share link

### PR DESCRIPTION
**Pull request by Erin007**
If the 2017 Minecraft tutorial is completed, the user will have a share link they can take to Minecraft Education to continue with their code.  This PR threads MCSharelink through to the 2017 Minecraft `VerticalImageResourceCard` if needed for both English and non-English. Also included are a few styling changes to accommodate the share link box with new copy. 

new mc sharelink

### CODE REVIEW

File: apps/src/sites/studio/pages/congrats/index.js [hunk 1]
```diff
@@ -17,10 +17,11 @@ $(document).ready(function () {
   ReactDOM.render(
     <Provider store={store}>
       <Congrats
-        completedTutorialType="other"
+        completedTutorialType="2017Minecraft"
         isRtl={isRtl}
         userType={userType}
         isEnglish={isEnglish}
+        MCShareLink="minecraft/sharelink"
```
Comment (Erin007) [inline apps/src/sites/studio/pages/congrats/index.js:24]: These are being set here to test that the MCShareLink is passed as expected.  Ultimately, completedTutorialType and MCShareLink will be pulled from the completed tutorial's url.

**Comment by Erin007** [comment]
cc: @tanyaparker

File: apps/src/templates/StudentsBeyondHoc.jsx [hunk 2]
```diff
@@ -50,6 +50,13 @@ export default class StudentsBeyondHoc extends Component {
     }
     const cards = cardSets[specificCardSet];
 
+// 2017 Minecraft Tutorial has a share link that can be used on Minecraft Education to import code.
+//Check if the 2017 Minecraft tutorial was completed;
+// if it was, update the Minecraft share link for the card that goes to Minecraft Education.
+    if (specificCardSet === "newMinecraftCards" && MCShareLink) {
```
Comment (breville) [inline apps/src/templates/StudentsBeyondHoc.jsx]: We just check for MCShareLink in the appropriate `case` block above.

File: apps/src/templates/StudentsBeyondHoc.jsx [hunk 3]
```diff
@@ -50,6 +50,13 @@ export default class StudentsBeyondHoc extends Component {
...
+      cards[2].MCShareLink = MCShareLink;
```
Comment (breville) [inline apps/src/templates/StudentsBeyondHoc.jsx]: Can we check for which card rather than assuming its index?

**Pull request merged**